circleci/config.go

```diff
@@ -0,0 +1,23 @@
+package circleci
```

Comment:
I was thinking maybe instead of removing the client completely, let it wrap (as a facade) the client from `github.com/jszwedko/go-circleci`. This way, the resource code would be less impacted and you wouldn't have to keep on doing things like `providerConfig.Client.WhateverMethod(**providerConfig.VcsType**, **providerConfig.Organization**, ...)` everywhere. What do you think?

Reply:
Sure, we can make a thin wrapper on top of `github.com/jszwedko/go-circleci` with ready to use functions.